apple-app-specific-password.md

```diff
@@ -1,13 +1,19 @@
 # Apple App-Specific Password
 
-The Apple app-specific password is a password for your Apple ID account that lets you sign in to your Apple account securely from a third-party app. It is required by the EAS Submission Service to submit your build to the Apple App Store.
+The Apple app-specific password is a password for your Apple ID account that lets you sign in to your Apple account securely from a third-party app. It is one of the authentication methods that can be used by the EAS Submit service to submit your build to the Apple App Store.
+
+### Security
+
+> [!WARNING]
+> This authentication method is considered less secure than using the App Store Connect API key by the Expo team, especially for personal Apple accounts, because of the broad permissions the app-specific password grants to the actor/service who has it. We recommend using the App Store Connect API key for submitting your builds to the Apple App Store (this is default and recommended authentication method). [Learn more](https://docs.expo.dev/app-signing/security/#apple-app-specific-password).
```

Comment:
```suggestion
> This authentication method is considered less secure than using the App Store Connect API key because of the broad permissions the app-specific password grants to the actor/service who has it. We recommend using the App Store Connect API key for submitting your builds to the Apple App Store (this is default and recommended authentication method). [Learn more](https://docs.expo.dev/app-signing/security/#apple-app-specific-password).
```